nextflow_schema.json

```diff
@@ -21,7 +19,7 @@
                     "pattern": "^\\S+\\.csv$",
                     "schema": "assets/schema_input.json",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with at least 4 columns, and a header row. See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
+                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/phyloplace/usage#samplesheet-input).",
```

Comment:
https://nf-co.re/phyloplace/usage#samplesheet-input shows 5 columns but doesnt tell whats required, as far as I see. And your sample sheet check seems to require 5 columns: https://github.com/nf-core/phyloplace/blob/722483575de6ba867e2671d0ca583675bcf56a8c/assets/schema_input.json#L46

Reply:
Thanks, forgot to update. Will do.